modules/appeals_api/spec/docs/notice_of_disagreements_spec.rb

```diff
@@ -152,7 +152,7 @@
       end
 
       response '404', 'Notice of Disagreement not found' do
-        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'errors', '404.json')))
```

Comment:
The `404.json` schema still contains an incorrectly nested `schema` key. I've worked around it by replacing it with `#/components/schemas/errorModel` in many places, but I still haven't managed to replace all of them. The other uses will still need to be fixed, either by removing the extra `schema` key in the json file itself or by switching to `#/components/schemas/errorModel` (I haven't updated the file itself because it would still introduce test failures at this point)